### PR TITLE
Linux dependencies in readme

### PR DESCRIPTION
**Pull request by electrocnic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,115 @@
 # Vulkan Launchpad Starter
 Starter template for a project based on Vulkan Launchpad
+
+
+
+# Linux
+
+## Dependencies
+
+### Ubuntu 22.04 (officially recommended by us)
```
Comment (johannesugb) [inline README.md:10]: Remove "officially recommended by us". Doesn't buy us anything, I guess ^^ Don't think that participants will start to install a new Linux?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,115 @@
...
+
+```
```
Comment (johannesugb) [inline README.md:12]: Maybe add a few lines of text what this does?!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,115 @@
...
+sudo apt clean all
+
+sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-11 60 --slave /usr/bin/g++ g++ /usr/bin/g++-11
+
+echo ""
+echo "Now running \"vulkaninfo\" to see if vulkan has been installed successfully:"
+vulkaninfo
+```
+
+### Ubuntu 20.04 (should work but we only tested the build in a docker environment, not if it runs!)
```
Comment (johannesugb) [inline README.md:36]:
```suggestion
### Ubuntu 20.04

_Note:_ Only tested in a Docker environment.

```

// Eigennamen immer schön groß schreiben!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,115 @@
...
+
+```
+sudo apt update && sudo apt upgrade -y && sudo apt install -y wget gpg git lsb-release software-properties-common && \
```
Comment (johannesugb) [inline README.md:39]: Maybe add a few lines of text what this does?!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,115 @@
...
+	sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test && \
+	sudo add-apt-repository -y ppa:oibaf/graphics-drivers && \
+	wget -qO- https://packages.lunarg.com/lunarg-signing-key-pub.asc | tee /etc/apt/trusted.gpg.d/lunarg.asc && \
+	wget -qO /etc/apt/sources.list.d/lunarg-vulkan-focal.list http://packages.lunarg.com/vulkan/lunarg-vulkan-focal.list && \
+	sudo apt update && sudo apt upgrade -y && sudo apt install -y g++ gdb make ninja-build rsync zip kitware-archive-keyring cmake libassimp-dev g++-11 libvulkan-dev libvulkan1 mesa-vulkan-drivers vulkan-tools vulkan-sdk dpkg-dev libvulkan1-dbgsym vulkan-tools-dbgsym libglfw3 libglfw3-dev libgl1-mesa-dev libglu1-mesa-dev libglew-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev && \
+	sudo apt clean all && \
+	sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-11 60 --slave /usr/bin/g++ g++ /usr/bin/g++-11
+```
+
+### Linux Mint 21.1 (should work but we only tested the build in a docker environment, not if it runs!)
```
Comment (johannesugb) [inline README.md:52]: same same (see above)

File: README.md [hunk 6]
```diff
@@ -1,2 +1,115 @@
...
+	sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test && \
+	sudo add-apt-repository -y ppa:oibaf/graphics-drivers && \
+	wget -qO- https://packages.lunarg.com/lunarg-signing-key-pub.asc | tee /etc/apt/trusted.gpg.d/lunarg.asc && \
+	wget -qO /etc/apt/sources.list.d/lunarg-vulkan-jammy.list http://packages.lunarg.com/vulkan/lunarg-vulkan-jammy.list && \
+	sudo apt update && sudo apt upgrade -y && sudo apt install -y  g++ gdb make ninja-build rsync zip kitware-archive-keyring cmake libassimp-dev g++-11 libvulkan-dev libvulkan1 mesa-vulkan-drivers vulkan-tools vulkan-sdk dpkg-dev libvulkan1-dbgsym vulkan-tools-dbgsym libglfw3 libglfw3-dev libgl1-mesa-dev libglu1-mesa-dev libglew-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev && \
+	sudo apt clean all && \
+	sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-11 60 --slave /usr/bin/g++ g++ /usr/bin/g++-11
+```
+
+### Debian Bullseye (should work but we only tested the build in a docker environment, not if it runs!)
```
Comment (johannesugb) [inline README.md:69]: same same (see above)

File: README.md [hunk 7]
```diff
@@ -1,2 +1,115 @@
...
+```
+git clone https://github.com/cg-tuwien/VulkanLaunchpadStarter.git && \
+	cd VulkanLaunchpadStarter && \
+	git checkout initialize-repo && \
+	mkdir out && cd out && \
+	cmake -DVulkan_INCLUDE_DIR=/VulkanTools/build/Vulkan-Headers/build/install/include/ -DVulkan_LIBRARY=/VulkanTools/build/Vulkan-Loader/build/install/lib/libvulkan.so -G Ninja .. && \
+	cmake --build . --config Debug
+```
+
+## Automatic git clone and build via commandline:
```
Comment (johannesugb) [inline README.md:105]: Please use title case for headings: https://titlecaseconverter.com/

**Review by johannesugb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (electrocnic) [reply README.md:10]: Naja ich hätt halt gern drauf aufmerksam gemacht welche fix funktionieren und welche auf eigene Gefahr gehen könnten.

**Pull request closed without merging**